### PR TITLE
Makefile sample update

### PR DESCRIPTION
**Pull request by rhoit**
just clean up the bit, sorry for late reply i don't really follow that email, since it is filled with spam.

### CODE REVIEW

**Comment by rhoit** [comment]
:tada: Thing is I can finally test it, `AppIndicator3` is no longer restricted to **Unity**, work with other **DE** :dancers:  too, You might have to do some **README** mods :pencil2:  .

But still couldn't make the schedule appear :grey_exclamation:  might be bug :bug: : .. can you check it i made few change in my first 2 commits.

P.S. tested in _XFCE_ :japanese_goblin:

**Comment by samundra** [comment]
@rhoit I tested the changes in Ubuntu14.04. It's great :) Thanks.

**Comment by samundra** [comment]
@rhoit 

After `sudo make install` I got the following output.

``` bash
rm -f /usr/local/bin/loadshedding
rm -rf /opt/loadshedding
mkdir -p /opt/loadshedding
install -m 755 main.py -t /opt/loadshedding/
cp README.md .version /opt/loadshedding/
ln -s /opt/loadshedding/main.py /usr/local/bin/loadshedding
```

but when I tried to execute loadshedding, I got following error

```
Traceback (most recent call last):
  File "/usr/local/bin/loadshedding", line 24, in <module>
    from nepali_calendar import NepaliDateConverter
ImportError: No module named nepali_calendar
```

How can I fix this?

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+SOURCES = main.py
```
Comment (rhoit) [inline Makefile:1]: Add required files here in `SOURCES` or in `SUPPORT` variables which are essential to execution
something like.

just my convention I keep executable stuff in `SOURCES`

```
SOURCES = main.py nepali_calendar.py preference.py convert.py
```

like you `date-converter.py` and `preference.py` files too..
:no_good: forgot to include this, I don't want to re-clone :sheep:.

Comment (samundra) [inline Makefile:1]: Thanks. Please PR :)

**Comment by samundra** [comment]
@rhoit I updated the Makefile as follows https://github.com/samundra/Nep_Loadshedding_Py3/blob/4bd9508465e168b622069180a82d55ccb2d7ab81/Makefile What do you think about it?

**Comment by rhoit** [comment]
that's too explicit, we normally don't write `if` `else` structure in `Makefile`. तिमिले त auto generated `Makefile` लाई पनि जित्यौ। :stuck_out_tongue_closed_eyes:

**Comment by samundra** [comment]
Was the previous one autogenerated ? For now its working fine :)

**Comment by rhoit** [comment]
no it wasn't.

**Pull request merged**